### PR TITLE
C_CPP :Lower triangular matrix

### LINKED ISSUE #3230: C_CPP : Lower and Upper Triangular Matrix
### Description

I will be explaining about how to create Lower Triangular and Upper Triangular Matrix using both row major and column major mapping.

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by kajaljha09**
<hr>

## Description 📜
Explained about Lower Triangular Matrix along with programs in C++ using both row major and column major mapping.
Fixes #3230

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: C_CPP/Data Structures/Matrix/Lower_triangular.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        cout<<endl;
+    }
+    DisplayC(m);
+    cout<<GetC(m,1,1);}
+}
+```
+
+Refrences :
+
+* [GeeksForGeeks](www.geeksforgeeks.org)
```
Comment (apeksha235) [inline C_CPP/Data Structures/Matrix/Lower_triangular.md:148]: The link that you've added do not go anywhere, so you can either remove them or edit them to the actual url of the page

**Review by apeksha235 — CHANGES_REQUESTED** [review]

File: C_CPP/Data Structures/Matrix/Lower_triangular.md (hunk 1 above)
Comment (kajaljha09) [reply C_CPP/Data Structures/Matrix/Lower_triangular.md]: > The link that you've added do not go anywhere, so you can either remove them or edit them to the actual url of the page

done @apeksha235

**Comment by kajaljha09** [comment]
@apeksha235 i did the suggested changes ,pls check

**Comment by kajaljha09** [comment]
@Suvraneel @Saviour1001  pls review this and consider it to merge .

**Comment by Suvraneel** [comment]
@kajaljha09  Veeeery good documentation...
image

Change the initial documentation a little

**Comment by Suvraneel** [comment]
@kajaljha09  LGTM ❤️ 🚀 :octocat:

**Pull request merged**